.tekton/source-to-image-rhel9-pull-request.yaml

```diff
@@ -0,0 +1,596 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift/source-to-image?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "master"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: source-to-image
+    appstudio.openshift.io/component: source-to-image-rhel9
+    pipelines.appstudio.openshift.io/type: build
+  name: source-to-image-rhel9-on-pull-request
+  namespace: ocp-tools-s2i-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/ocp-tools-s2i-tenant/source-to-image-rhel9:on-pr-{{revision}}
+  - name: image-expires-after
+    value: 5d
+  - name: dockerfile
+    value: Dockerfile.rhel9
+  pipelineSpec:
+    description: |
```

Comment:
We need use the remote-pipeline pattern here.
https://konflux-ci.dev/docs/patterns/keep-remote-pipelines-up-to-date/

Our pipelines are here in this repo. Use this pipeline for multiarch build.
https://github.com/redhat-openshift-builds/release/blob/main/pipelines/konflux-build-multi-platform.yaml

This change required for rhel8 pipelines also.